content/en/Integrations/webhooks.md

```diff
@@ -36,7 +36,7 @@ You can configure webhooks through the [API](https://docs.cobalt.io/v2/#webhooks
 
 ## Before You Start
 
-Before you start creating webhooks, complete the configuration in your app.
+Before you start creating webhooks, you'll need to complete configuration in the place receiving these updates.
```

Comment:
@nktrk 

> you'll need to complete

We use present tense in our documentation. Many trustworthy style guides, such as [Google Developer Documentation Style Guide](https://developers.google.com/style/tense), discourage the use of future tenses:

https://developers.google.com/style/tense

> the place receiving these updates

IMO, _your app_ is self-explanatory.

Alternative option:

in the app where you want to receive update